src/material-examples/dialog-content/dialog-content-example-dialog.html

```diff
@@ -0,0 +1,7 @@
+<h2 md-dialog-title>Delete all</h2>
+<md-dialog-content>Are you sure?</md-dialog-content>
+<md-dialog-actions>
+  <button md-button tabindex="-1">Not Tabbable</button>
+  <button md-button [md-dialog-close]="true" tabindex="1">Yes</button>
```

Comment:
A more realistic example for this case would be to have an input and have the `md-dialog-close` pass the input value back to the `afterClosed`.